css/stocktools/gui.scss

```diff
@@ -262,4 +262,8 @@ $button-hover-color: #e6ebf5;
   width: 50%;
   height: 20px;
   float: left;
-}
\ No newline at end of file
+}
+
+li.highcharts-disabled-btn:hover, .highcharts-disabled-btn .highcharts-submenu-item-arrow:hover {
+  background-color: #f7f7f7;
```

Comment:
Use the `$button-background-color` variable instead. Makes theming easier more predictable.